src/main/java/picard/util/IntervalListTools.java

```diff
@@ -458,7 +465,17 @@ protected int doWork() {
         }
 
         LOG.info("Produced " + intervalCount + " intervals totalling " + totalBaseCount + " bases.");
-        OUTPUT_VALUE.output(totalBaseCount, intervalCount);
+        if (COUNT_OUTPUT != null) {
+            IOUtil.assertFileIsWritable(COUNT_OUTPUT);
```

Comment:
these asserts should be added to customCommandLineValidation

Reply:
I suppose the `assertFileIsWritable` could be in `customCommandLineValidation`, though this is inconsistent with the treatment of other inputs/outputs.  All the assertions on readability/writability of `INPUT`, `SECOND_INPUT`, and `OUTPUT` are done in the `doWork()` method.  I am happy to leave them where they are, or move them all to `customCommandLineValidation`, just think it should be consistent.

Reply:
as long as it happens in the beginning of the invocation, rather than just before writing out, I'm fine.

Reply:
got it, moved to beginning of `doWork()`